src/views/Languages/index.jsx

```diff
@@ -0,0 +1,297 @@
+      width: `calc(100% - ${theme.drawerWidth}px)`,
+    },
+  },
+  title: {
+    ...theme.mixins.gutters(),
+  },
+  button: {
+    color: theme.palette.common.white,
+    '& svg': {
+      fill: theme.palette.common.white,
+    },
+  },
+}))
+export default class Languages extends Component {
+  state = {
+    drawerOpen: false,
+    error: null,
+  };
+
+  handleDrawerToggle = () => {
+    this.setState({ drawerOpen: !this.state.drawerOpen });
+  };
+
+  handleBugInfoClick = memoizeWith(
+    id => id,
+    async id => {
+      try {
+        const {
+          data: { comments },
+        } = await this.props.client.query({
+          query: commentsQuery,
+          variables: { id },
+          context: { client: 'bugzilla' },
+        });
+
+        return comments[0].text;
+      } catch (error) {
+        this.setState({ error });
+      }
+    }
+  );
+
+  render() {
+    const {
+      classes,
+      match: {
+        params: { language },
+      },
+      bugzilla: bugzillaData,
+    } = this.props;
+    const { drawerOpen, error } = this.state;
+    const drawer = (
+      <Fragment>
+        <div className={classes.drawerHeader}>
+          <Typography variant="title">Skills</Typography>
+          <Hidden mdUp>
+            <IconButton onClick={this.handleDrawerToggle}>
+              <CloseIcon />
+            </IconButton>
+          </Hidden>
+        </div>
+        <Divider light />
+        <Sidebar activeItem={language} onItemClick={this.handleDrawerToggle} />
+      </Fragment>
+    );
+    const goodFirstBugs =
+      (bugzillaData &&
+        bugzillaData.goodFirst &&
+        uniqBy(
+          bugzillaData.goodFirst.edges.map(edge => edge.node).map(bug => ({
+            assignee: bug.status === 'ASSIGNED' ? bug.assignedTo.name : '-',
+            project: bug.component,
+            tags: [
+              ...(bug.keywords || []),
+              ...extractWhiteboardTags(bug.whiteboard),
+            ],
+            summary: bug.summary,
+            lastUpdated: bug.lastChanged,
+            id: bug.id,
+            url: `https://bugzilla.mozilla.org/show_bug.cgi?id=${bug.id}`,
+          })),
+          'summary'
+        )) ||
+      [];
+    const mentoredBugs =
+      (bugzillaData &&
+        bugzillaData.mentored &&
+        uniqBy(
+          bugzillaData.mentored.edges.map(edge => edge.node).map(bug => ({
+            assignee: bug.status === 'ASSIGNED' ? bug.assignedTo.name : '-',
+            project: bug.component,
+            tags: [
+              ...(bug.keywords || []),
+              ...extractWhiteboardTags(bug.whiteboard),
+            ],
+            summary: bug.summary,
+            lastUpdated: bug.lastChanged,
+            id: bug.id,
+            url: `https://bugzilla.mozilla.org/show_bug.cgi?id=${bug.id}`,
+          })),
+          'summary'
+        )) ||
+      [];
+
+    return (
+      <div className={classes.root}>
+        <AppBar position="fixed" className={classes.header}>
```

Comment:
Similar to the project view, we want a back arrow button that will navigate the user back to the main view.